tests/unit/test_cspConfig.js

```diff
@@ -1,96 +0,0 @@
-import require from 'require-uncached';
-
-const deployedEnvs = ['dev', 'production', 'stage'];
-
-const cdnHosts = {
-  dev: 'https://addons-dev-cdn.allizom.org',
-  stage: 'https://addons-stage-cdn.allizom.org',
-  production: 'https://addons.cdn.mozilla.net',
-};
-
-const apiHosts = {
-  dev: 'https://addons-dev.allizom.org',
-  stage: 'https://addons.allizom.org',
-  production: 'https://addons.mozilla.org',
-};
-
-describe(__filename, () => {
-  const existingNodeEnv = process.env.NODE_ENV;
-
-  afterEach(() => {
-    process.env.NODE_ENV = existingNodeEnv;
-  });
-
-  for (const env of deployedEnvs) {
-    // eslint-disable-next-lint no-loop-func
-    it(`should have a source-list config for ${env}`, () => {
-      process.env.NODE_ENV = env;
-      // Reset the require cache so that the config require
-      // takes into account changes to NODE_ENV.
-      jest.resetModules();
-      const cdnHost = cdnHosts[env];
-      const apiHost = apiHosts[env];
-      const config = require('config');
-      const cspConfig = config.get('CSP').directives;
-      expect(cspConfig.scriptSrc).toContain(cdnHost);
-      expect(cspConfig.scriptSrc).not.toContain("'self'");
-      expect(cspConfig.imgSrc).toContain(cdnHost);
-      expect(cspConfig.imgSrc).toContain("'self'");
-      expect(cspConfig.styleSrc).toContain(cdnHost);
-      expect(cspConfig.styleSrc).not.toContain("'self'");
-      expect(cspConfig.connectSrc).toContain(apiHost);
-      expect(cspConfig.connectSrc).not.toContain("'self'");
-    });
-  }
-
-  describe('CSP defaults', () => {
```

Comment:
I believe we don't have these test cases anymore. We test the config here, which isn't something we usually do, but I can understand why we want to make sure CSP are covered by tests.